tests/test-start-stop

```diff
@@ -3,15 +3,70 @@
 
 # Minimal mptcpd start/stop test.
 #
-# Copyright (c) 2018, 2019, Intel Corporation
+# Copyright (c) 2018, 2019, 2021, Intel Corporation
 
+# Non-interactive sudo capable if exit code is 0.
+echo Running non-interactive sudo check...
+sudo -n echo "    succeeded"
+sudo_exit_code=$?
 
-set -e
+variable=`sysctl net.mptcp | grep enabled | sed -e 's/\(net.mptcp.*enabled\).*$/\1/'`
+old_value=`sysctl $variable | sed -e 's/.*= \(.*\)$/\1/'`
+
+disable_value=0
+enable_value=1
+
+# "Disabled" value is always 0, but the "enabled" value could be 1 or
+# 2, depending on the kernel in use.  For example, the
+# multipath-tcp.org kernel "enabled" value could be 1 or 2.  Use the
+# old "enabled" value in case it is 2 instead of 1.
+if [ $old_value -ne 0 ]
```

Comment:
My system complains here:

`tests/test-start-stop: line 23: [: too many arguments`

Reply:
Fixed by commit ef905e7.

The regex didn't correctly handle multiple `net.mptcp` sysctl variables with `enabled` in them.  In particular, the 5.13 kernel on my Ubuntu box only has `net.mptcp.enabled` whereas the 5.15 kernel, for example, also has a `net.mptcp.checksum_enabled` variable, resulting multiple values being assigned to the `$old_value` shell variable.